CHANGES.rst

```diff
@@ -1,10 +1,12 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-09)
 --------------------------
 
 - Allows ``htcondor_max_runtime`` and ``htcondor_accounting_group`` to be specified for HTC jobs.
+- Fixes docker build, exiting when there are problems with ``myschedd`` installation.
```

Comment:
- See my comments in https://github.com/reanahub/reana-workflow-engine-yadage/pull/170/files#r519923812 regarding HTC and CI phrases.

- Fixes Docker build by properly exiting when there are problems with `myschedd` installation.